### DISCUSSION TITLE
Off by 1 minzoom?

### DISCUSSION THREAD

**Discussion by four43** [osm2pgsql-dev/osm2pgsql#2281] [Themepark]
When constructing a themepark theme, I have the following conditional for my way:
```lua
if a.minzoom < 10 then -- XXX TODO some kind of off-by-one error here?
    themepark:insert('railways_low', a, t)
end
```
I see a bunch of ways in my railways_low table that are of minzoom 10... There are no other references to this in my theme. I'm very curious about that "some kind of off-by-one error" here line? This seems impossible and I'm just as confused as that comment.

Thanks for the excellent project.

**Comment by joto** [comment]
That comment is in a few places in the Shortbread configs. I don't remember why I wrote that, but apparently I thought that there might be a problem. I, or somebody else, has to go through there in detail and look at the code and the Shortbread spec and figure out what's what.

**Reply by four43** [reply]
I believe I ran into the same issue, it was very odd. My `railways_low` table is full of minzooms of 10. I had to go double check Lua didn't do a silly <= automatically or something like their index start of 1 :P
